README.md

```diff
@@ -53,6 +61,7 @@ This README documents the current strawman architecture.  Changes must be made v
     1. Non-KP, Non-ARA components, such as normalizers, must also be registered and provide metadata appropriate to their API type.
     1. The SmartAPI Registry will provide a unified query system, returning information about all three API methods.  This query system will allow ARAs to locate the appropriate KPs.
     1. SRI will guarantee that metadata standards across the components allow such a unified query system.
+    1. The SmartAPI registry will allow components to find all KPs by querying for biolink predicates. The SmartAPI registry will allow components to query by predicate from biolink-understood vocabularies, and return KPs that provide such metadata.
```

Comment:
How will this query work with super-predicates and sub-predicates? For example:

1. a query for related_to could return KPs that provide has_phenotype
2. a query for has_phenotype could return KPs that provide related_to

I'm not sure which one(s) of these are desirable.

Reply:
This is an excellent question,  and one that we will discuss on todays (11/3) Architecture call.

That said, I think that we can consider this PR as being for exact matches and then update with more detail once that discussion is concluded.

Reply:
The semantic web community have put a lot of thought into this; see https://www.w3.org/TR/sparql11-entailment/ cc @micheldumontier @vemonet @balhoff